src/app/pages/classes-page/classes-page.component.html

```diff
@@ -0,0 +1,49 @@
+<section class="background-image">
+    <div class = "content-page">
+      <div class="selectStats">
+        <h1 class="title">Statistiques </h1>
+        <h2>Répartissez vos {{points}} points </h2>
+
+        <ul>
+        @for(stat of statistiques; track stat)
+        {
+          <li>
+            <div class="stat">
+              <div class="statName">{{stat.name}}</div> 
+              <div class="statScore">{{stat.score}}</div>
+            </div>
+          </li>
+        }
+        </ul>
+        
+      </div>
+
+      <div class="selectClass">
+          <h1 class="title">Sélection des classes</h1>
```

Comment:
only one H1 per page

Reply:
This file must not appear in that PR but i take into account your requests